### ISSUE TITLE
Error messages are not announced for inputs.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#482]
# Fields, Banners

Visibility: 3
Evaluation: Partially Supports
Success Criteria: 3.3.3: Error Suggestion
Needs design: 

### Description
Error messages are often included within the child element of an input’s label. However, VoiceOver with Safari on macOS only reports the first text node. As an alternative the error message can be moved out of the <label> element and associated with the input using [aria-describedby](https://www.w3.org/TR/wai-aria-1.1/#aria-describedby). In the future [aria-errormessage](https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage) can be used, but as of now it isn’t well supported. 

When a validation error occurs, the input should be marked with [aria-invalid](https://www.w3.org/TR/wai-aria-1.1/#aria-invalid).

![]()

**Comment by amazingphilippe** [comment]
This is linked to PR [#1242](https://github.com/cds-snc/notification-admin/pull/1242) and should be fixed QA'd and merged already!
